portfolio/src/main/webapp/map-script.js

```diff
@@ -0,0 +1,157 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var map;
+var journeyCoordinates = [];
+
+/**
+ * Initial the map and add it to the map page.
+ */
+function initMap() {
+    var mapCenter = { lat: 30.168979, lng: -171.859749 }; // Somewhere in North Pacific Ocean
```

Comment:
Should be `var mapCenter = {lat: 30.168979, lng: -171.859749};` with no space next to '{' and '}'.
https://google.github.io/styleguide/jsguide.html#formatting-array-literals

Reply:
The empty spaces are removed.